Networking/Networking/Model/OrderFeeLine.swift

```diff
@@ -5,7 +5,13 @@ import Codegen
 ///
 public struct OrderFeeLine: Equatable, Codable, GeneratedFakeable, GeneratedCopiable {
     public let feeID: Int64
-    public let name: String
+
+    /// Fee Name
+    ///
+    /// Sending a null value to the REST API removes the Fee Line from the Order.
+    ///
+    public let name: String?
```

Comment:
Yes, it really isn't intuitive 😬 

> I wonder if we should create some kind of OrderFeeLineFactory and expose a deletedLine method or similar to constraint this "hack" into a single place.

That's a good idea, especially since it isn't obvious/intuitive. I'll take a look at that before merging this one.

Reply:
@Ecarrion What do you think of putting a method for this in the existing `Yosemite.OrderFactory` (rather than a new `OrderFeeLineFactory`), since `OrderFeeLine` is only used in the context of an `Order`? Similar to how we have a `simplePaymentFee` method there to create a simple payment fee line, we can have a `deletedLine` or `deletedFee` method to create a fee line that deletes the fee from the order.

I also realize that we should do something similar to help with deleting shipping lines. I'll go ahead and merge this PR and then work on a separate PR for this work.